blueprints/app/files/bower.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
let me know if i misunderstood this comment thread, but it sounds like:

* @nathanhammond is saying: using a warning a user would get when using `bower` cli directly themselves
*  @Turbo87 is saying: when ember cli invokes `bower`, it handles this case already.

If so, it seems reasonable to me that a user intending to once again use bower, is warned to first use `bower init`.